src/main/resources/kibana-rest.yml

```diff
@@ -62,3 +62,43 @@ kibana_user:
   versions:
     ">= 7.6.0": "/internal/security/me"
     ">= 5.0.0 < 7.6.0": "/api/security/v1/me"
+
+kibana_security_exception_list:
+  versions:
+    ">= 7.10.0": "/api/exception_lists/_find?namespace_type=agnostic"
```

Comment:
The call being added to this PR will only fetch namespace agnostic exception lists, which in most situations will only include a single "endpoint" exception list. For this reason, could we also add the following call? This will return all of the exception lists associated with the detection rules.

```
"/api/exception_lists/_find?namespace_type=single&per_page=1000"
```